### PR TITLE
fix: buttonColor on hovered

### PR DESCRIPTION
**Pull request by jiyuujin**
背景色を修正

### CODE REVIEW

File: lib/widgets/cfs_button.dart [hunk 1]
```diff
@@ -23,7 +23,7 @@ class CfsButton extends StatelessWidget {
             shape: const StadiumBorder(),
             padding: const EdgeInsets.all(24),
             primary: Colors.deepOrange,
-            onPrimary: Colors.white54,
+            onPrimary: Colors.black87,
           ),
           child: Text(
```
Comment (shcahill) [inline lib/widgets/cfs_button.dart:28]: This `Text` widget has `backgroundColor` (deepOrange) and it hide the ripple effect, I think 🤔 

image

Comment (shcahill) [reply lib/widgets/cfs_button.dart:28]: (sorry for confusing issue description...)

Comment (jiyuujin) [reply lib/widgets/cfs_button.dart:28]: Should have changed evenly by another commit

Comment (shcahill) [reply lib/widgets/cfs_button.dart:28]: Thanks for the fix.
LGTM

**Review by shcahill — APPROVED** [review]

**Pull request merged**